src/app.py

```diff
@@ -151,9 +151,10 @@ def appium_run(avd_name: str):
             selenium_port = int(os.getenv('SELENIUM_PORT', 4444))
             selenium_timeout = int(os.getenv('SELENIUM_TIMEOUT', 30))
             selenium_proxy_class = os.getenv('SELENIUM_PROXY_CLASS', 'org.openqa.grid.selenium.proxy.DefaultRemoteProxy')
+            application_name = os.getenv('APPLICATION_NAME', avd_name)
```

Comment:
maybe we should rename it to SELENIUM_APPLICATION_NAME, so we know that this is used for capability thing. and you should also add it in the README_APPIUM_AND_SELENIUM.md